hub/core/storage/shuffle_lru_cache.py

```diff
@@ -15,20 +15,20 @@ def __init__(
         next_storage: Optional[StorageProvider],
         cache_size: int,
         dataset,
-        num_workers: int,
         tensor_keys: Optional[Sequence[str]],
         transform: Optional[Callable],
         mode: Optional[str] = None,
+        num_workers: int = 2
```

Comment:
you should only specify these default worker values in the API level functions.

also, why did you choose num workers 2? wouldn't it be better to default to 1?

Reply:
yes I am removing them and modifying only  the `pytorch` , `tensor` and `compute` parts